crates/bevy_pbr/src/render/mesh_view_bindings.wgsl

```diff
@@ -49,4 +49,5 @@
 @group(0) @binding(17) var depth_prepass_texture: texture_depth_2d;
 @group(0) @binding(18) var normal_prepass_texture: texture_2d<f32>;
 @group(0) @binding(19) var motion_vector_prepass_texture: texture_2d<f32>;
+@group(0) @binding(20) var deferred_prepass_texture: texture_2d<u32>;
```

Comment:
Currently in the engine yes, it's only used by deferred_lighting. But it would be used by a lot of things in the future like GI, Decals, etc... It's also important to have a separate `normal_prepass_texture` rather than needing to use the full `deferred_prepass_texture` since reads of just the normals will be substantially faster than reads of the whole gbuffer. Not sure what the best solution is.

Reply:
I wonder if we will hit the limits based on declared bindings, or used bindings…

Reply:
good question... there are fallback bindings so they are always bound, but if they are not used in the shader they may not count.

Reply:
Seems like something we could test somehow. Just spam a bunch of unused bindings? Do unused bindings have to be bound?

Reply:
Anyway, sounds like it's a useful texture to have bound.